cdap-docs/admin-manual/source/_includes/installation/upgrading.txt

```diff
@@ -0,0 +1,93 @@
+.. _distribution-upgrading:
+
+==============================================================
+|hadoop-distribution|: Upgrading an Existing CDAP Installation
+==============================================================
+
+When upgrading an existing CDAP installation from a previous version, you will need
+to make sure the CDAP table definitions in HBase are up-to-date.
+
+These steps will stop CDAP, update the installation, run an upgrade tool for the table definitions,
+and then restart CDAP.
+
+These steps will upgrade from CDAP 3.1.x to 3.2.x. If you are on an earlier version of CDAP,
+please follow the upgrade instructions for the earlier versions and upgrade first to 3.1.x before proceeding.
+
+.. highlight:: console
+
+1. Stop all flows, services, and other programs in all your applications.
+
+#. Stop all CDAP processes::
+
+     $ for i in `ls /etc/init.d/ | grep cdap` ; do sudo service $i stop ; done
+
+#. Update the CDAP file definition lists by running either of these methods:
+ 
+   - Using Yum:
+
+     .. include:: ../../_includes/installation/installation.txt 
+        :start-after: Download the Cask Yum repo definition file:
+        :end-before:  .. end_install-rpm-using-yum
+
+   - Using APT:
+
+     .. include:: ../../_includes/installation/installation.txt 
+        :start-after: Download the Cask APT repo definition file:
+        :end-before:  .. end_install-debian-using-apt
+
+#. Update the CDAP packages by running either of these methods:
+
+   - Using Yum (on one line)::
+
+       $ sudo yum install cdap cdap-gateway \
+             cdap-hbase-compat-0.96 cdap-hbase-compat-0.98 cdap-hbase-compat-1.0 \
+             cdap-hbase-compat-1.0-cdh cdap-hbase-compat-1.1 \
+             cdap-kafka cdap-master cdap-security cdap-ui
+
+   - Using APT (on one line)::
+
+       $ sudo apt-get install cdap cdap-gateway \
+             cdap-hbase-compat-0.96 cdap-hbase-compat-0.98 cdap-hbase-compat-1.0 \
+             cdap-hbase-compat-1.0-cdh cdap-hbase-compat-1.1 \
+             cdap-kafka cdap-master cdap-security cdap-ui
+
+#. If you are upgrading a secure Hadoop cluster, you should authenticate with ``kinit``
+   before the next step (running the upgrade tool)::
```

Comment:
as in the next step, it should be clear that this step is to be run as the CDAP user